packages/message-db/message-db/test/snapshot.test.ts

```diff
@@ -0,0 +1,53 @@
+import { Codec, ITimelineEvent } from "@equinox-js/core"
+import { randomUUID } from "crypto"
+import { test, expect } from "vitest"
+import * as Snapshot from "../src/lib/Snapshot.js"
+import * as Token from "../src/lib/Token.js"
+
+test("Properly loads snapshot version from previous schema", async () => {
+  const event: ITimelineEvent = {
+    id: randomUUID(),
+    index: 0n,
+    isUnfold: true,
+    size: 0,
+    time: new Date(),
+    type: "Snapshotted",
+    data: JSON.stringify({ hello: "world" }),
+    meta: JSON.stringify({ streamVersion: "0" }),
```

Comment:
should this be something other that -1/null/0/1 (or should there be specific tests for that?
aoso probably make act and assert pieces DRY?